### DISCUSSION TITLE
NPM problem installing Next JS App With Termux on Android

### DISCUSSION THREAD

**Discussion by giofanny** [vercel/next.js#34284] [Help] [answered]
Specifications:
- Node Js LTS v 16.14.0 
- Device Realme C2*
- 10 Android **OS** 
- No Root
Using Termux v 0.117 Latest
Creating a new Next.js app in /data/data/com.termux/files/home/**nextjs-blog**

/* separator */
**Input dan Output**

$ npm init next-app nextjs-blog
Creating a new Next.js app in /data/data/com.termux/files/home/nextjs-blog.

Using npm.

Installing dependencies:
- react
- react-dom
- next

npm ERR! code ERR_SOCKET_TIMEOUT
npm ERR! network Socket timeout
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /data/data/com.termux/files/home/.npm/_logs/2022-02-13T03_41_39_592Z-debug-0.log

Aborting installation.
  npm install --save-exact --save react react-dom next has failed.

$ node -v
v16.14.0

$

**Comment by giofanny** [comment, marked as answer]
Oh calm I was able to outsmart it by merging the VPN app, And now I've successfully installed Nextjs App on Android Thanks and running npm run dev.

**Reply by mimiza** [reply]
How did you do it. Please share
